cmd/metricchecker/cmd.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/PingCAP-QE/metrics-checker/pkg/metric"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "Metrics checker",
+	Short: "For checking prometheus metrics",
+	Run: func(cmd *cobra.Command, args []string) {
+		InitConfig(configFilePath, configBase64)
+
+		address = metric.AddHTTPIfIP(address)
+		if grafanaAPIURL != "" {
+			dashboardName := "Metrics Checker"
+			grafanaAPIURL = metric.AddHTTPIfIP(grafanaAPIURL)
+			err := metric.CreateMetricsDashboard(grafanaAPIURL, dashboardName, config.MetricsToShow)
+			if err != nil {
+				log.Fatalf("Create grafana metrics error: %s", err)
+			}
+			log.Printf("Created dashboard %s on %s", dashboardName, grafanaAPIURL)
```

Comment:
`log.Debug/Info/Warn/Error` is preferred.

Reply:
As the `metrics-checker` is a command line tools, I think we can directly use `cmd.Println`.
We can use `log.xxx` in pkg/metrics.

Reply:
I think we can disable log by default. For example, go test only output logs when `-v` option is specified.

Reply:
Well... I got it. We shouldn't print log in `/pkg/metric` since it's a package. Package shouldn't print logs.